source/nodejs/adaptivecards-site/package.json

```diff
@@ -3,13 +3,15 @@
 	"version": "0.9.0-alpha.0",
 	"private": true,
 	"scripts": {
+		"build": "npm run generate && npm run generate-sw",
 		"clean": "hexo clean",
-		"build": "hexo generate && npm run generate-sw",
-		"release-deps": "npx lerna run --scope @microsoft/adaptivecards-site --include-dependencies release --stream",
 		"generate-md-ac": "node node_modules/marked-schema/bin/marked-schema ../../../schemas/adaptive-card.json ./schema-explorer-toc.yml -r AdaptiveCard",
 		"generate-md-hc": "node node_modules/marked-schema/bin/marked-schema ../../../schemas/host-config.json ./schema-hostconfig-toc.yml -r AdaptiveCardConfig",
-		"release": "hexo clean && hexo generate && npm run generate-sw",
+		"release": "npm run generate && npm run generate-sw",
```

Comment:
shouldn't we add `hexo clean` before  `npm run generate`?

Reply:
hmm... let me look into this :)